src/sass/textbox/textbox.scss

```diff
@@ -244,4 +244,8 @@ textarea.textbox__control[readonly]:focus {
 
 .textbox--fluid {
     width: 100%;
+
+    & .textbox__control {
```

Comment:
Can we do 
```
.textbox--fluid,
.textbox--fluid .textbox__control {
```
instead?
This way its easier to read in CSS inspector where the styles are coming from initially.